oasis-webapp/src/main/resources/templates/Login.soy

```diff
@@ -120,11 +124,10 @@ aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-d
         {/if}
         <div class=btn-row>
           <button>
-            <img alt="" src="/images/go.png" width=50 height=50>
             {msg desc="Form submit button"}
               Log in
             {fallbackmsg desc="Form submit button"}
-              Go!
+              Go !
```

Comment:
Ah, ok, didn't know for the space.

And ... I don't know why I did that ... I knew it is a translatable content ...